go-controller/pkg/ovn/egressgw_test.go

```diff
@@ -2190,6 +2190,10 @@ var _ = ginkgo.Describe("OVN Egress Gateway Operations", func() {
 								Name: types.GWRouterPrefix + nodeName,
 								UUID: types.GWRouterPrefix + nodeName + "-UUID",
 							},
+							&nbdb.LogicalSwitch{
```

Comment:
Had to add LS to the unit tests, because syncNode expects to find an LS for each node.
I'm assuming that is okay, yes?

Reply:
yep np